index.html

```diff
@@ -0,0 +1,300 @@
+                      © 2021 NEWSWEEK DIGITAL LLC
+                      </span>
+                </div>
+                <div>
+                  <i class="fab fa-facebook-square fa-2x"></i>
+                  <i class="fab fa-twitter fa-2x"></i>
+                  <i class="fab fa-instagram fa-2x"></i>
+                  <i class="fab fa-tumblr fa-2x"></i>
+                  <i class="fab fa-linkedin-in fa-2x"></i>
+                  
+                  <!-- 
+                  icons from font awesome will go here, the image cant be here --> 
+                   
+                </div>
+            </div>
+
+          <div class="border-top navbar-light">
+            <ul class="navbar-nav  d-flex flex-row p-1 footer-links">
+              <li class="nav-item font-weight-bold">
+                <a class="nav-link text-decoration-none" href="#">Editions:</a>
+              </li>
+              <li class="nav-item border-right p-0 pr-2 pl-2">
+                <a class="nav-link" href="#">U.S. Edition</a>
+              </li>
+              <li class="nav-item border-right p-0 pr-2 pl-2">
+                <a class="nav-link" href="#">日本</a>
+              </li>
+              <li class="nav-item border-right p-0 pr-2 pl-2">
+                <a class="nav-link" href="#">Pakistan</a>
+              </li>
+              <li class="nav-item border-right p-0 pr-2 pl-2">
+                <a class="nav-link" href="#">Polska</a>
+              </li>
+              <li class="nav-item p-0 pr-2 pl-2">
+                <a class="nav-link" href="#">România</a>
+              </li>
+            </ul>
+          </div>
+    
+
+            <div class="navbar-expand-lg navbar-light justify-content-between text-secondary">
+            <ul class="nav footer-links">
+              <li class=" border-right px-1 ">
+                About Us
+              </li>
+              <li class=" border-right px-1 ">
+                Corrections
+              </li>
+              <li class=" border-right px-1 ">
+                Contact Us
+              </li>
+              <li class="border-right px-1">
+                Editorial Guidelines
+              </li>
+              <li class=" border-right px-1 ">
+                Advertise
+              </li>
+              <li class=" border-right px-1 ">
+                Copyright
+              </li>
+              <li class=" border-right px-1 ">
+                Terms & Conditions
+              </li>
+              <li class=" border-right px-1 ">
+                Privacy Policy
+              </li>
+              <li class=" border-right px-1 ">
+                Cookie Policy
+              </li>
+              <li class=" border-right px-1">
+                Terms of Sale
+              </li>
+              <li class=" border-right px-1 ">
+                Archive
+              </li>
+              <li class="  px-2 ">
+                Announcements
```

Comment:
- [ ] Use semantic HTML for links. `<a>` anchor tag

Reply:
done